### PR TITLE
clean up docs

### PR DESCRIPTION
**Pull request by htor**
* be more consistent.
* remove støy
* add type info on arguments (drop table listing of options and args, it's too verbose)
* add `opener` to `core-dialog` https://github.com/nrkno/core-components/issues/187

### CODE REVIEW

File: packages/core-input/readme.md [hunk 1]
```diff
@@ -79,31 +79,29 @@ Typing toggles the [hidden attribute](https://developer.mozilla.org/en/docs/Web/
...
 ### React / Preact
 
-<small><b>Important:</b> First direct of `<CoreToggle>` must be a `<input>` and next direct child  will be used for results</small>
-
 ```js
 import CoreInput from '@nrk/core-input/jsx'
 
 // All props are optional, and defaults are shown below
 // Props like className, style, etc. will be applied as actual attributes
-// <Input> will handle state itself unless you call event.preventDefault() in onFilter or onSelect
```
Comment (htor) [inline packages/core-input/readme.md]: skal det stå `<CoreInput>` her?

File: packages/core-input/readme.md [hunk 2]
```diff
@@ -79,31 +79,29 @@ Typing toggles the [hidden attribute](https://developer.mozilla.org/en/docs/Web/
...
+// <Input> will handle state itself unless you call event.preventDefault() in onToggle
```
Comment (eirikbacker) [inline packages/core-input/readme.md]: should still be `onFilter`, `onSelect` or `onAjax` (onAjax is new)

File: packages/core-input/readme.md [hunk 3]
```diff
@@ -62,13 +58,17 @@ Typing toggles the [hidden attribute](https://developer.mozilla.org/en/docs/Web/
 
 ## Usage
 
+Typing toggles the [hidden attribute](https://developer.mozilla.org/en/docs/Web/HTML/Global_attributes/hidden) on items of type `<button>` and `<a>`, based on matching [textContent](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent). Focusing the input unhides the following element.
+The default filtering behavior can easily be altered through the `'input.select'` and `'input.filter'` [events](#events).
```
Comment (eirikbacker) [inline packages/core-input/readme.md]:
```suggestion
The default filtering behavior can easily be altered through the `'input.select'`, `'input.filter'`, `'input.ajax'` and  `'input.ajax.beforeSend'` [events](#events).
```

File: packages/core-progress/readme.md [hunk 4]
```diff
@@ -81,20 +81,17 @@ demo-->
...
 coreProgress(
-  selector, // Selector string or HTML Element
-  value // See table below
-);
+  String|Element|Elements,  // Accepts a selector string, NodeList, Element or array of Elements
+  Number|String|Object      // Optional. Set value, indeterminate status or options
+)
+
+// Examples:
+coreProgress('.my-progress')                         // Initalize tabs on element
```
Comment (eirikbacker) [inline packages/core-progress/readme.md]:
```suggestion
coreProgress('.my-progress')                         // Initalize and ensure accessibility correct attributes
```

File: packages/core-tabs/readme.md [hunk 5]
```diff
@@ -87,14 +87,33 @@ demo-->
...
 coreTabs(
   String|Element|Elements,    // Accepts a selector string, NodeList, Element or array of Elements
-  open                        // Optional. Can be String: id of tab, Element: tab or Number: index of tab
+  String|Element|Number       // Optional. Set tab selection with string id, element or tab index number
 )
+
+// Examples:
+coreTabs('.my-tabs')              // Initalize tabs on element
+coreTabs('.my-tabs', 3)           // Initalize tabs and select 3rd tab
+coreTabs('.my-tabs', '#tab-2')    // Initalize tabs and select 2nd tab
```
Comment (eirikbacker) [inline packages/core-tabs/readme.md]: Also add example with dom element as 2nd attribute? :)

File: packages/core-toggle/readme.md [hunk 6]
```diff
@@ -44,6 +43,106 @@ Content is only toggled when clicking `button`. Great for accordions and expand/
...
+.my-toggle {}                         /* Target button in any state */
+.my-toggle[aria-expanded="true"] {}   /* Target only open button */
+.my-toggle[aria-expanded="false"] {}  /* Target only closed button */
+
+.my-toggle-content {}                 /* Target content in any state */
+.my-toggle-content:not([hidden]) {}   /* Target only open content */
+.my-toggle-content[hidden] {}         /* Target only closed content */
+```
+
 ## Demo: Popup
```
Comment (eirikbacker) [inline packages/core-toggle/readme.md]: Why is demo popup moved after everything? Popup is one of the most used features, so was therefore put before everything earlier

Comment (htor) [reply packages/core-toggle/readme.md]: for consitency, we have one demo on the top, usage after that ... and rest of demos before FAQ or similar to the end. the important part is that usage is not all the way in the middle of the document.

File: packages/core-input/readme.md (hunk 3 above)
Comment (htor) [reply packages/core-input/readme.md]: yap

File: packages/core-tabs/readme.md (hunk 5 above)
Comment (htor) [reply packages/core-tabs/readme.md]: 2nd argument? line 116 does that, no?

File: packages/core-toggle/readme.md (hunk 6 above)
Comment (eirikbacker) [reply packages/core-toggle/readme.md]: 👍 would suggest popup-example in top then :)

**Pull request merged**